DateRangeBox: add code snippets and extended description #5043 (#5111)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDateRangeBox/1 Configuration/multiView.md

---
---
##### shortDescription

> Specifies whether the UI component displays multiple views that allows you to pick a date range. 

Reviewer: I'm not sure why the option or description uses the word "multiple". From what I saw in demos, this is the option that switches between one-month view and two-month view, right? 

```suggestion
Specifies whether the UI component displays a two-month calendar in the drop-down. 
```

Author: Yes. But in the nearest future developers are going to add more views, for instance 4-calendar and more. There will be another option to manage count of views, but "multiView" will specify whether the UI component displays one or multiple views that allows you to pick a date range.

Reviewer: Will this work then? 

```suggestion
Specifies whether the UI component displays a single-month calendar or a multi-month calendar. 
```

"displays multiple views" doesn't work in any case. The component can support multiple views... But what it displays at one time is just a single view... that view may display a single-month calendar, or a full-year calendar, or something else. But "multiple views" is not the right term.
